# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/blend/reftest.list
# Commit: 431c5142ff3a
# Full Hash: 431c5142ff3ad102328d90cb29470b20ccd1271e
# Author: Glenn Watson <github@intuitionlibrary.com>
# Date: 2019-06-13 21:50:57
# Regressor Bug: 1558106
# File Overlap Count: 1
# Description:
#   Bug 1558106 - Support picture caching for multiple scroll roots. r=kvark
#   
#   This patch implements the majority of the planned picture caching
#   improvements. It supports most of the functionality required to
#   (as a follow up) support OS compositor integration. It also improves
# ==============================================================================

diff -r 855d557b82a2 -r 431c5142ff3a gfx/wr/wrench/reftests/blend/reftest.list
--- a/gfx/wr/wrench/reftests/blend/reftest.list	Wed Jun 12 16:18:22 2019 +0000
+++ b/gfx/wr/wrench/reftests/blend/reftest.list	Thu Jun 13 04:43:56 2019 +0000
@@ -6,7 +6,7 @@
 
 skip_on(android,debug) == multiply.yaml multiply-ref.yaml
 skip_on(android) == multiply-2.yaml multiply-2-ref.yaml
-skip_on(android) == color_targets(3) alpha_targets(0) multiply-3.yaml multiply-2-ref.yaml
+skip_on(android) == color_targets(4) alpha_targets(0) multiply-3.yaml multiply-2-ref.yaml
 skip_on(android) == difference.yaml difference-ref.yaml
 skip_on(android) fuzzy(1,30000) == difference-transparent.yaml difference-transparent-ref.yaml
 skip_on(android) == darken.yaml darken-ref.yaml
@@ -17,7 +17,7 @@
 == isolated.yaml isolated-ref.yaml
 skip_on(android) == isolated-2.yaml isolated-2-ref.yaml
 == isolated-with-filter.yaml isolated-ref.yaml
-== isolated-premultiplied.yaml blank.yaml
+skip_on(android,emulator) == isolated-premultiplied.yaml blank.yaml
 == isolated-premultiplied-2.yaml isolated-premultiplied-2-ref.yaml
 
 == large.yaml large-ref.yaml